.drone.yml

```diff
@@ -25,72 +25,105 @@ platform:
   arch: amd64
 
 trigger:
+  branch:
+    - main
   event:
     - push
 
 steps:
-  - name: build & push main
+  - name: build & push safe
     image: plugins/docker
+    depends_on:
+      - clone
     settings:
       registry: quay.io
       repo: quay.io/ukhomeofficedigital/hocs-toolbox
+      target: safe
       tags:
         - build_${DRONE_BUILD_NUMBER}
         - ${DRONE_COMMIT_SHA}
+        - ${DRONE_COMMIT_SHA}-safe
         - latest
     environment:
       DOCKER_PASSWORD:
         from_secret: QUAY_ROBOT_TOKEN
       DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
-    when:
-      branch:
-        include:
-          - main
 
-  - name: build & push
+  - name: build & push scary
+    depends_on:
+      - clone
     image: plugins/docker
     settings:
       registry: quay.io
       repo: quay.io/ukhomeofficedigital/hocs-toolbox
+      target: dangerous
       tags:
-        - build_${DRONE_BUILD_NUMBER}
-        - ${DRONE_COMMIT_SHA}
+        - ${DRONE_COMMIT_SHA}-dangerous
     environment:
       DOCKER_PASSWORD:
         from_secret: QUAY_ROBOT_TOKEN
       DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
-    when:
-      branch:
-        exclude:
-          - main
 
 ---
 
 kind: pipeline
 type: kubernetes
 name: deploy
```

Comment:
Can we use the tagging mechanism with the GitHub actions and use that way for this? That way we get SemVer for free even if it's unused at this point. 

Prefer to be consistent, is all.